### PR TITLE
Update getstarted.html.erb

### PR DESCRIPTION
**Pull request by edoardo123-PAY**
https://payments-platform.atlassian.net/browse/PP-12321

This is the changes which have been raised by XL Bully around the confusion in who can use which PSP provider

### CODE REVIEW

File: source/getstarted.html.erb [hunk 1]
```diff
@@ -132,21 +132,19 @@ title: Get started
                 You must have a Payment Service Provider (PSP) to process the payments taken using GOV.UK&nbsp;Pay. However, there is no procurement work required to obtain a PSP.
                 This saves you time and money, and you’re not locked into a long-term contract.
               </p>
-              <p class="govuk-body">There are 2 PSP options available:</p>
+              <p class="govuk-body">The PSP you will use depends on the type of organisation you work for:</p>
```
Comment (NathanD-GDS) [inline source/getstarted.html.erb:135]: I'd change `you will` to `you'll`, in line with style guide advice on ctractions.

Comment (NathanD-GDS) [reply source/getstarted.html.erb:135]:
```suggestion
              <p class="govuk-body">The PSP you'll use depends on the type of organisation you work for:</p>
```

**Review by hjvoid — APPROVED** [review]
👌

**Comment by iqbalgds** [comment]
Closing - replacing with this: https://github.com/alphagov/pay-product-page/pull/718

**Pull request closed without merging**